src/Entities/CarEntity.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace KoalaCars\Entities;
+
+use KoalaCars\Abstracts\VehicleAbstract;
+
+class CarEntity extends VehicleAbstract
+{
+    private int $id;
+    private string $make;
+    private string $model;
+    private int $year;
+    private string $color;
+    private string $location;
+    private string $image;
+
+    public function getId(): int
+    {
+       return $this->id;
+    }
+    public function getMake(): string
+    {
+       return $this->make;
+    }
+    public function getModel(): string
+    {
+       return $this->model;
+    }
+    public function getYear(): int
```

Comment:
i think im right in saying some of the years are empty? so this could return null? therefore this should be a nullable return type